budget_proj/requirements.txt

```diff
@@ -11,3 +11,4 @@ requests==2.13.0
 six==1.10.0
 
 gunicorn
+mixer==5.6.6
```

Comment:
I am still confused as to why there are two `requirements.txt` files, but does this dependency need to be added to the other one also?